ceagle/api/client.py

```diff
@@ -51,12 +60,19 @@ def get(self, uri="/", **kwargs):
 
 
 def get_client(service_name):
-    """Return client for given service anme, if possible.
+    """Return client for given service name, if possible.
 
     :param service_name: str name of microservice
     :returns: Client
     """
+
+    if config.get_config().get("use_fake_api_data", True):
+        fake_client_class = FAKE_CLIENT_MAP.get(service_name)
+        if fake_client_class:
+            return fake_client_class(name=service_name, endpoint="_fake_")
```

Comment:
So this will return a fake client even if service_name is absent in config? Thi slooks like ignoring config options. I think both real & fake client should be returned only if service appears in config - this is checked at L74-76